src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -0,0 +1,89 @@
+import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
+import { useEffect, useMemo, useState } from 'react';
+import { Pie } from 'react-chartjs-2';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+interface statistics {
+  waiting_time: number,
+  waiting_time_title: string,
+  waiting_time_count: number,
+  waiting_time_percent: number
+}
+
+interface WaitingTimeStatisticsProps {
+  slug: string;
```

Comment:
The `slug` prop in `WaitingTimeStatisticsProps` seems to be declared but not used.